packages/web3-core-method/src/index.js

```diff
@@ -890,11 +892,10 @@ function _handleTxPricing(method, tx) {
                         maxFeePerGas = tx.gasPrice;
                         delete tx.gasPrice;
                     } else {
-                        maxPriorityFeePerGas = tx.maxPriorityFeePerGas || '0x9502F900'; // 2.5 Gwei
+                        maxPriorityFeePerGas = tx.maxPriorityFeePerGas || method.defaultMaxPriorityFeePerGas
```

Comment:
users can always override default with maxPriorityFeePerGas by explicitly providing it via options and also there is no activity on this PR so closing it.